src/commands/template.rs

```diff
@@ -26,6 +26,10 @@ pub struct TemplateCommand {
     #[clap(skip)]
     #[builder(default, setter(into, strip_option))]
     build_id: Option<Uuid>,
+
+    #[clap(skip)]
+    #[builder(default)]
+    os_version: String,
```

Comment:
This won't work when you use the `template` command. This will end up being an empty string and cause the template to be useless if a user is wanting to use the akmods module in their recipe.